vevestaX/vevesta.py

```diff
@@ -19,10 +19,12 @@ class Experiment(object):
     def __init__(self):
         self.__dataSourcing = None
         self.__featureEngineering = None
-        self._data=None
+        self.__data=None
     
-        self.startlocals = None
+        self.__primitiveDataTypes = [int, str, float, bool]
+        self.__startlocals = None
         self.__variables = {}
+        self.get_filename()
```

Comment:
make a variable self.__filename = self.getFilename()...and make changes through out the code where self.get_filename is used.